### DISCUSSION TITLE
Single Reed Bagpipe optimization

### DISCUSSION THREAD

**Discussion by CamStLouis** [edwardkort/WWIDesigner#131] [Reed Study] [answered]
Trying to re-do hole layout and length on a historical bagpipe chanter, to bring it in line with my preferred half-closed fingering. Unlike the more well-known bagpipes out there, this kind has a single reed - essentially a drone reed plugged into the chanter. Is the "reed study" mode still the best way to approach this, even though it's designed for double reeds?

**Comment by bhp1** [comment, marked as answer]
The reed model is not specific to double-reeds.  It should work with single-reeds and even lip-reed instruments.  The reed model is still experimental at this point.  One question that isn't answered for single-reeds: what is the reference point for the "mouthpiece position"?  You may need to try different values for the mouthpiece position to see what gives the best calibration results.

**Reply by CamStLouis** [reply]
Cool. I imagine it's actually the end of the air column contained in the reed. 

Guess we'll experiment and see what works!
